### DISCUSSION TITLE
Any way to bidirectional sync labels?

### DISCUSSION THREAD

**Discussion by Nalci** [hcengineering/platform#6072] [Q&A] [answered]
I'm new to huly and love it so far but I can't find a way to sync labels from huly to github or the other way around. Is it not possible (yet)?

**Comment by aplatoff** [comment]
Hi @haiodo, please can you shed some light on this? Is there serious reasons why we do not support labels sync for now?

**Comment by haiodo** [comment, marked as answer]
Hi, Labels synchronisation are not yet implemented, so no way to sync them for now. 
Once it will be we will sync from Huly for github labels and vice verse.
